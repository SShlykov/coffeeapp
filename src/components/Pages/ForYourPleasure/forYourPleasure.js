import React, {Component} from "react";
import coffee from "./img/CoffeeShade.jpg";
import {BeansLogo, Item} from "../../SmallComp";

export default class ForYourPleasure extends Component {
    render() {
        let imageSrc = "https://images-na.ssl-images-amazon.com/images/I/71qBQnpQFYL._SL1500_.jpg"
        return(
            <section className="shop">
                <div className="container">
                    <div className="row">
                        <div className="col-lg-4 offset-lg-2">
                            <img className="shop__girl" src={coffee} alt="girl"/>
                        </div>
                        <div className="col-lg-4">
                            <div className="title">About our goods</div>
                            <BeansLogo dark/>
                            <div className="shop__text">
                                Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                                <br/>
                                <br/>
                                Afraid at highly months do things on at. Situation recommend objection do intention<br/>
                                so questions.<br/>
                                As greatly removed calling pleased improve an. Last ask him cold feel<br/>
                                met spot shy want. Children me laughing we prospect answered followed. At it went<br/>
                                is song that held help face.<br/>
                            </div>
                        </div>
                    </div>
                    <div className="line"> </div>
                    <div className="row">
                        <div className="col-lg-10 offset-1">
                            <div className="shop__wrapper">
                                <Item name="AROMISTICO Coffee" weight="1" country="Brazil" price="6.99" imgSrc={imageSrc}/>
                                <Item name="AROMISTICO Coffee" weight="1" country="Kenya" price="6.99" imgSrc={imageSrc}/>
                                <Item name="AROMISTICO Coffee" weight="1" country="Columbia" price="6.99" imgSrc={imageSrc}/>
                                <Item name="AROMISTICO Coffee" weight="1" country="Brazil" price="6.99" imgSrc={imageSrc}/>
                                <Item name="AROMISTICO Coffee" weight="1" country="Brazil" price="6.99" imgSrc={imageSrc}/>
                                <Item name="AROMISTICO Coffee" weight="1" country="Brazil" price="6.99" imgSrc={imageSrc}/>
                            </div>
                        </div>
                    </div>
                </div>
            </section>
        )
    }
}