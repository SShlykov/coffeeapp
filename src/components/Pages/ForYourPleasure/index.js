import ForYourPleasure from "./forYourPleasure";
export default ForYourPleasure;