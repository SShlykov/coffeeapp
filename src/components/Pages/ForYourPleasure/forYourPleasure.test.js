import React from 'react';
import ForYourPleasure from "./forYourPleasure";
import {shallow} from 'enzyme';

describe("Testing <ForYourPleasure />", () => {
    const page = shallow(<ForYourPleasure/>);
    describe("Testing snap & state", ()=>{
        it("ForYourPleasure has rendered correctly", () => {
            expect(page).toMatchSnapshot();
        })
    })
})