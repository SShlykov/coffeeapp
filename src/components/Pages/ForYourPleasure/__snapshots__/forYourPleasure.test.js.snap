// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <ForYourPleasure /> Testing snap & state ForYourPleasure has rendered correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ForYourPleasure />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-lg-4 offset-lg-2"
          >
            <img
              alt="girl"
              className="shop__girl"
              src="CoffeeShade.jpg"
            />
          </div>
          <div
            className="col-lg-4"
          >
            <div
              className="title"
            >
              About our goods
            </div>
            <BeansLogo
              dark={true}
            />
            <div
              className="shop__text"
            >
              Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
              <br />
              <br />
              Afraid at highly months do things on at. Situation recommend objection do intention
              <br />
              so questions.
              <br />
              As greatly removed calling pleased improve an. Last ask him cold feel
              <br />
              met spot shy want. Children me laughing we prospect answered followed. At it went
              <br />
              is song that held help face.
              <br />
            </div>
          </div>
        </div>
        <div
          className="line"
        >
           
        </div>
        <Items />
      </div>,
      "className": "shop",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="row"
          >
            <div
              className="col-lg-4 offset-lg-2"
            >
              <img
                alt="girl"
                className="shop__girl"
                src="CoffeeShade.jpg"
              />
            </div>
            <div
              className="col-lg-4"
            >
              <div
                className="title"
              >
                About our goods
              </div>
              <BeansLogo
                dark={true}
              />
              <div
                className="shop__text"
              >
                Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                <br />
                <br />
                Afraid at highly months do things on at. Situation recommend objection do intention
                <br />
                so questions.
                <br />
                As greatly removed calling pleased improve an. Last ask him cold feel
                <br />
                met spot shy want. Children me laughing we prospect answered followed. At it went
                <br />
                is song that held help face.
                <br />
              </div>
            </div>
          </div>,
          <div
            className="line"
          >
             
          </div>,
          <Items />,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="col-lg-4 offset-lg-2"
              >
                <img
                  alt="girl"
                  className="shop__girl"
                  src="CoffeeShade.jpg"
                />
              </div>,
              <div
                className="col-lg-4"
              >
                <div
                  className="title"
                >
                  About our goods
                </div>
                <BeansLogo
                  dark={true}
                />
                <div
                  className="shop__text"
                >
                  Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                  <br />
                  <br />
                  Afraid at highly months do things on at. Situation recommend objection do intention
                  <br />
                  so questions.
                  <br />
                  As greatly removed calling pleased improve an. Last ask him cold feel
                  <br />
                  met spot shy want. Children me laughing we prospect answered followed. At it went
                  <br />
                  is song that held help face.
                  <br />
                </div>
              </div>,
            ],
            "className": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="girl"
                  className="shop__girl"
                  src="CoffeeShade.jpg"
                />,
                "className": "col-lg-4 offset-lg-2",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "girl",
                  "className": "shop__girl",
                  "src": "CoffeeShade.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="title"
                  >
                    About our goods
                  </div>,
                  <BeansLogo
                    dark={true}
                  />,
                  <div
                    className="shop__text"
                  >
                    Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                    <br />
                    <br />
                    Afraid at highly months do things on at. Situation recommend objection do intention
                    <br />
                    so questions.
                    <br />
                    As greatly removed calling pleased improve an. Last ask him cold feel
                    <br />
                    met spot shy want. Children me laughing we prospect answered followed. At it went
                    <br />
                    is song that held help face.
                    <br />
                  </div>,
                ],
                "className": "col-lg-4",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "About our goods",
                    "className": "title",
                  },
                  "ref": null,
                  "rendered": "About our goods",
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "dark": true,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      "Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.",
                      <br />,
                      <br />,
                      "Afraid at highly months do things on at. Situation recommend objection do intention",
                      <br />,
                      "so questions.",
                      <br />,
                      "As greatly removed calling pleased improve an. Last ask him cold feel",
                      <br />,
                      "met spot shy want. Children me laughing we prospect answered followed. At it went",
                      <br />,
                      "is song that held help face.",
                      <br />,
                    ],
                    "className": "shop__text",
                  },
                  "ref": null,
                  "rendered": Array [
                    "Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "Afraid at highly months do things on at. Situation recommend objection do intention",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "so questions.",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "As greatly removed calling pleased improve an. Last ask him cold feel",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "met spot shy want. Children me laughing we prospect answered followed. At it went",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    "is song that held help face.",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                  ],
                  "type": "div",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " ",
            "className": "line",
          },
          "ref": null,
          "rendered": " ",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "data": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-lg-4 offset-lg-2"
            >
              <img
                alt="girl"
                className="shop__girl"
                src="CoffeeShade.jpg"
              />
            </div>
            <div
              className="col-lg-4"
            >
              <div
                className="title"
              >
                About our goods
              </div>
              <BeansLogo
                dark={true}
              />
              <div
                className="shop__text"
              >
                Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                <br />
                <br />
                Afraid at highly months do things on at. Situation recommend objection do intention
                <br />
                so questions.
                <br />
                As greatly removed calling pleased improve an. Last ask him cold feel
                <br />
                met spot shy want. Children me laughing we prospect answered followed. At it went
                <br />
                is song that held help face.
                <br />
              </div>
            </div>
          </div>
          <div
            className="line"
          >
             
          </div>
          <Items />
        </div>,
        "className": "shop",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="row"
            >
              <div
                className="col-lg-4 offset-lg-2"
              >
                <img
                  alt="girl"
                  className="shop__girl"
                  src="CoffeeShade.jpg"
                />
              </div>
              <div
                className="col-lg-4"
              >
                <div
                  className="title"
                >
                  About our goods
                </div>
                <BeansLogo
                  dark={true}
                />
                <div
                  className="shop__text"
                >
                  Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                  <br />
                  <br />
                  Afraid at highly months do things on at. Situation recommend objection do intention
                  <br />
                  so questions.
                  <br />
                  As greatly removed calling pleased improve an. Last ask him cold feel
                  <br />
                  met spot shy want. Children me laughing we prospect answered followed. At it went
                  <br />
                  is song that held help face.
                  <br />
                </div>
              </div>
            </div>,
            <div
              className="line"
            >
               
            </div>,
            <Items />,
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-lg-4 offset-lg-2"
                >
                  <img
                    alt="girl"
                    className="shop__girl"
                    src="CoffeeShade.jpg"
                  />
                </div>,
                <div
                  className="col-lg-4"
                >
                  <div
                    className="title"
                  >
                    About our goods
                  </div>
                  <BeansLogo
                    dark={true}
                  />
                  <div
                    className="shop__text"
                  >
                    Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                    <br />
                    <br />
                    Afraid at highly months do things on at. Situation recommend objection do intention
                    <br />
                    so questions.
                    <br />
                    As greatly removed calling pleased improve an. Last ask him cold feel
                    <br />
                    met spot shy want. Children me laughing we prospect answered followed. At it went
                    <br />
                    is song that held help face.
                    <br />
                  </div>
                </div>,
              ],
              "className": "row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    alt="girl"
                    className="shop__girl"
                    src="CoffeeShade.jpg"
                  />,
                  "className": "col-lg-4 offset-lg-2",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "girl",
                    "className": "shop__girl",
                    "src": "CoffeeShade.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <div
                      className="title"
                    >
                      About our goods
                    </div>,
                    <BeansLogo
                      dark={true}
                    />,
                    <div
                      className="shop__text"
                    >
                      Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.
                      <br />
                      <br />
                      Afraid at highly months do things on at. Situation recommend objection do intention
                      <br />
                      so questions.
                      <br />
                      As greatly removed calling pleased improve an. Last ask him cold feel
                      <br />
                      met spot shy want. Children me laughing we prospect answered followed. At it went
                      <br />
                      is song that held help face.
                      <br />
                    </div>,
                  ],
                  "className": "col-lg-4",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "About our goods",
                      "className": "title",
                    },
                    "ref": null,
                    "rendered": "About our goods",
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "dark": true,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.",
                        <br />,
                        <br />,
                        "Afraid at highly months do things on at. Situation recommend objection do intention",
                        <br />,
                        "so questions.",
                        <br />,
                        "As greatly removed calling pleased improve an. Last ask him cold feel",
                        <br />,
                        "met spot shy want. Children me laughing we prospect answered followed. At it went",
                        <br />,
                        "is song that held help face.",
                        <br />,
                      ],
                      "className": "shop__text",
                    },
                    "ref": null,
                    "rendered": Array [
                      "Extremity sweetness difficult behaviour he of. On disposal of as landlord horrible.",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "Afraid at highly months do things on at. Situation recommend objection do intention",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "so questions.",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "As greatly removed calling pleased improve an. Last ask him cold feel",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "met spot shy want. Children me laughing we prospect answered followed. At it went",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                      "is song that held help face.",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {},
                        "ref": null,
                        "rendered": null,
                        "type": "br",
                      },
                    ],
                    "type": "div",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": " ",
              "className": "line",
            },
            "ref": null,
            "rendered": " ",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "data": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
